sql/catalyst/src/test/scala/org/apache/spark/sql/connector/catalog/CatalogSuite.scala

```diff
@@ -111,6 +111,29 @@ class CatalogSuite extends SparkFunSuite {
     assert(catalog.tableExists(testIdent))
   }
 
+  test("createTable: partitioned table") {
+    val partCatalog = new InMemoryPartitionTableCatalog
+    partCatalog.initialize("test", CaseInsensitiveStringMap.empty())
+
+    assert(!partCatalog.tableExists(testIdent))
+
+    val columns = Array(
+        Column.create("col0", IntegerType),
+        Column.create("part0", IntegerType))
+    val table = partCatalog.createTable(
```

Comment:
If you want to add a test coverage for `createTable` method, could you revise the PR title in that way?

Reply:
Yeah, that's what it means.
The original intention was that a colleague asked me yesterday how to create `a partitioned table` using `TableCatalog#createTable`.  I reviewed the code and found that this explicit usage method was missing from the UT.
Because the concept of `Transform` is very obscure, it is not easy for `new guys` to understand how to use it.